packages/styles/scss/components/filter-panel/_filter-panel.scss

```diff
@@ -134,7 +134,8 @@
 
     &:not([has-selections]) {
       .#{$prefix}--clear {
-        display: none;
+        color: $ibm-color__gray-40;
```

Comment:
I think we probably have a better color token to use here. Maybe @oliviaflory knows?

Reply:
My guess would be `$disabled-02`. Given that the background of the filter panel is Grey 10 (`$ui-01`), and there are tokens for `$disabled-n`. The guidance states that you need to use `$disabled-02` on top of `$disabled-01`, which is Grey 10 as well. 

https://v10.carbondesignsystem.com/guidelines/color/usage

![image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/assets/191049/4e8cc3a8-d284-43a4-8d7b-a3a8d3ada890)

Reply:
Hi @andy-blum and @m4olivei 
I agree to use the color as per described in carbon guidelines, thought I find it a bit too light in contrast with the background, I think it works.